Explorer/Assets/Scripts/MVC/IWindowsStackManager.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace MVC
+{
+    public interface IWindowsStackManager
+    {
+        IController TopMostPopup { get; }
+
+        /// <summary>
+        ///     For each sortingLayer there is a separate stack with a begin value for <see cref="CanvasOrdering.OrderInLayer" />
+        /// </summary>
+        PopupPushInfo PushPopup(IController controller);
+
+        FullscreenPushInfo PushFullscreen(IController controller);
+
+        void PopFullscreen(IController controller);
+
+        PersistentPushInfo PushPersistent(IController controller);
+
+        TopPushInfo PushTop(IController controller);

```

Comment:
Can we find a different naming for top-most UIs. Maybe "overlay"? `TopPushInfo`/`PushTop` sound strange...